app/controllers/searches_controller.rb

```diff
@@ -5,7 +5,7 @@ def new
     @search = Search.new(query: params[:query]) || Search.new
     @query = []
     @query << params[:query]
-    @results = ["actor1", "actor2"]
+    @results = ["actor1", "actor2", "actor3", "actor4"]
   end
 
```

Comment:
i think you can just delete your searches controller, because it still has my okd code and will probably crash gergs new version :)

Reply:
Done, thanks for the hint.